circle.yml

```diff
@@ -3,20 +3,20 @@ machine:
     LANG: en_US.UTF-8
     LC_CTYPE: en_US.UTF-8
   xcode:
-    version: "6.2"
+    version: "6.3.1"
 dependencies:
   pre:
-    - xcrun instruments -w "iPhone 6 (8.2 Simulator)" || exit 0
+    - xcrun instruments -w "iPhone 6 (8.3 Simulator)" || exit 0
   override:    
-    - sudo gem install cocoapods xcpretty obcd slather -N
-    - pod install --project-directory=Example
+    - bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4
```

Comment:
Good to know, will remove. Any idea why this is [failing to build](https://circleci.com/gh/splinesoft/SSDataSources/55) with `The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.` ?

The script does run `pod install` immediately before building. This might be new with 0.37?